### ISSUE TITLE
Continue Listening is showing books that have been completed (100%)

### ISSUE THREAD

**Issue by mdeeter** [advplyr/audiobookshelf-web#60]
Web docker version 2.5.0

Expectation... if a book is completed/finished, then it should not be displayed in the "Continue Listening" section.

![image](https://github.com/advplyr/audiobookshelf-web/assets/136902/6b608f27-cc2e-403f-b63d-9c44c81734b0)

![image](https://github.com/advplyr/audiobookshelf-web/assets/136902/5b9e75e4-63ed-4bb7-8acd-e53750ed0e14)

Bonus issue:

When I finished "Blood Rites" (book 6), I would have expected to see "Dead Beat" (book 7) in the "Continue Series" section, but it did not display there.

**Comment by advplyr** [comment]
This has been tracked down to have to do with a downloaded book on mobile for an older version creating a duplicate media progress row in the database. If you still have the book downloaded on mobile you may be able to open it and discard progress to resolve the issue. Otherwise this would require manually editing the database unfortunately.

See https://github.com/advplyr/audiobookshelf-app/issues/931
and https://github.com/advplyr/audiobookshelf/issues/1983#issuecomment-1783864608

Also, this is the wrong repository to open issues in. advplyr/audiobookshelf for server and website, advplyr/audiobookshelf-app for mobile app issues.

**Comment by mdeeter** [comment]
Note that this isn't using the mobile app... But the website.

**Comment by advplyr** [comment]
Yeah it doesn't matter. If you used the mobile app at any point for those books and they were downloaded then it could have created the duplicate media progress row in the db.

**Comment by mdeeter** [comment]
Thanks for the clarification.
